### DISCUSSION TITLE
Correcting false positives with new Coral TPU setup

### DISCUSSION THREAD

**Discussion by jbeck22** [blakeblackshear/frigate#8267] [General]
I've used Frigate in the past, but I recently rebuilt my setup and now I'm using the USB Coral TPU.  I have it enabled in my config:
`detectors:
  coral:
    type: edgetpu
    device: usb`

But I'm getting a LOT of false positives at the moment.  Do I need to do anything else with models or should the TPU be better at this already?

![image](https://github.com/blakeblackshear/frigate/assets/4844115/7bed6bfd-8342-4058-9c89-27db6cf3352b)

![image](https://github.com/blakeblackshear/frigate/assets/4844115/d10b5651-f4a9-4d94-9241-14d1632c7570)

Here's my config file...
[https://pastebin.com/CW36SFSG](url)

**Comment by NickM-27** [comment]
The coral just runs models faster, the accuracy and precision of the detection is entirely based on the model. 

Frigate offers many filters to circumvent these, like min_area, max_area, min_ratio, etc 

This is explained more https://docs.frigate.video/guides/false_positives

The default model is trained on the COCO dataset which looks nothing like security cameras. 

Otherwise, there is Frigate+ which is a service that lets you train a model fine tuned for your cameras on top of a base model trained on camera examples.

https://deploy-preview-6262--frigate-docs.netlify.app/plus/
